scripts/exgfs_atmos_awips_20km_1p0deg.sh

```diff
@@ -75,7 +77,7 @@ echo "#######################################"
 echo " Process GRIB AWIP GRIB2 PRODUCTS      "
 echo "#######################################"
 echo " "
-set -x
+${TRACE_ON:-set -x}
```

Comment:
What is the use of the preamble if every script will set/reset this.

Reply:
This is only used if the script is using `set +x` before this point, usually to echo a bunch of lines (the wave scripts do a lot of this). This makes sure after that section has completed, it returns to its prior state instead of blinding turning `set -x` back on. This way if you turn `TRACE=NO`, then it won't turn the trace on here.